packages/blade/src/components/Input/DropdownInputTriggers/AutoComplete.tsx

```diff
@@ -127,7 +127,8 @@ const useAutoComplete = ({
         name: props.name,
         value: displayText,
       });
-      setInputValue(displayText ?? '');
+      // Use displayText as inputValue only if its not controlled by user
+      setInputValue((props.value as string) ?? displayText ?? '');
```

Comment:
displayText is supposed to be different from value.

E.g. if `value="mumbai"`, and we have that value mapped to `"Mumbai"` in ActionListItem, then the value displayed in input is "Mumbai" and not "mumbai".


I think what we need to do here might be to not call `setInputValue` when value is being controlled 🤔

Reply:
Let me check if that fixes the issue.